sim-lib/src/lib.rs

```diff
@@ -1127,12 +1122,17 @@ impl PaymentResultLogger {
     }
 }
 
+// TODO: Add activity name and handle the initial processing log with few LOC
 impl Display for PaymentResultLogger {
     fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
         let total_payments = self.success_payment + self.failed_payment;
+
+        // if self.activity_name
```

Comment:
Incomplete comment and TODO left in this commit?

Reply:
Also unaddressed?

Reply:
Then the [fixup should be squashed](https://jordanelver.co.uk/blog/2020/06/04/fixing-commits-with-git-commit-fixup-and-git-rebase-autosquash/) as requested.

Reply:
Ok. Thank you.